src/main/java/ratpack/pac4j/internal/RatpackWebContext.java

```diff
@@ -20,40 +20,54 @@
 import com.google.common.collect.Maps;
 import com.google.common.collect.Sets;
 import io.netty.handler.codec.http.cookie.DefaultCookie;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
 import org.pac4j.core.context.Cookie;
 import org.pac4j.core.context.WebContext;
 import org.pac4j.core.context.session.SessionStore;
-import org.pac4j.core.exception.HttpAction;
+import org.pac4j.core.exception.TechnicalException;
+import org.pac4j.core.exception.http.HttpAction;
+import org.pac4j.core.profile.ProfileManager;
 import ratpack.exec.Promise;
 import ratpack.form.Form;
 import ratpack.form.internal.DefaultForm;
 import ratpack.form.internal.FormDecoder;
 import ratpack.handling.Context;
-import ratpack.http.*;
+import ratpack.http.HttpMethod;
+import ratpack.http.MediaType;
+import ratpack.http.Request;
+import ratpack.http.Response;
+import ratpack.http.TypedData;
 import ratpack.server.PublicAddress;
 import ratpack.session.Session;
 import ratpack.session.SessionData;
 import ratpack.util.MultiValueMap;
 
-import java.net.URI;
-import java.util.*;
-
```

Comment:
Ordering/expanding of wildcard imports.  While in general I think it's good practice to use single imports for reproducibility, that doesn't seem to be a practice that this project currently has in place.